community/main.py

```diff
@@ -206,50 +205,225 @@ def update(self, delta_time):
+		super().__init__(window)
+		self.texture_manager = texture_manager.TextureManager(0, 0, 0)
+
+		self.icon = self.texture_manager.load_texture("dirt")
+		self.logo = self.texture_manager.load_texture("logo")
+
+	def on_draw(self):
+		super().on_draw()
+		gl.glEnable(gl.GL_TEXTURE_2D)
+		self.window.clear()
+
+	def update_ui(self):
+		super().update_ui()
+		io = imgui.get_io()
+		imgui.set_next_window_size(io.display_size.x, io.display_size.y)
+		imgui.set_next_window_position(0, 0)
+		imgui.push_style_var(imgui.STYLE_WINDOW_BORDERSIZE, 0.0)
+		imgui.push_style_var(imgui.STYLE_WINDOW_PADDING, (0.0, 0.0))
+
+		if imgui.begin(
+			"MC PY",
+			True,
+			flags=
+			imgui.WINDOW_NO_DECORATION |
+			imgui.WINDOW_ALWAYS_AUTO_RESIZE |
+			imgui.WINDOW_NO_SAVED_SETTINGS |
+			imgui.WINDOW_NO_NAV
+		):
+			# Get the draw list
+			draw_list = imgui.get_window_draw_list()
+
+			# Tile the image across the window
+			for x in range(0, int(io.display_size.x), self.icon[1]):
+				for y in range(0, int(io.display_size.y), self.icon[2]):
+					draw_list.add_image(self.icon[0].id, (x, y), (x + self.icon[1], y + self.icon[2]))
+
+			# Draw a semi-transparent overlay to darken the background
+			overlay_color = imgui.get_color_u32_rgba(0, 0, 0, 0.75)
+			draw_list.add_rect_filled(0, 0, io.display_size.x, io.display_size.y, overlay_color)
+
+
+			# Calculate the position to horizontally center the image
+			image_width = self.logo[1] / 2.5
+			image_height = self.logo[2] / 2.5
+
+			# Set the cursor position to the calculated center position
+			imgui.set_cursor_pos(((io.display_size.x - image_width) / 2, 50))
+
+			# Draw the image at the calculated position
+			imgui.image(self.logo[0].id, image_width, image_height, (0, 1), (1, 0))
+
+			# Calculate the position to horizontally center the buttons
+			button_width = 300  # Adjust button width as needed
+			button_height = 30  # Adjust button height as needed
+			button_x = (io.display_size.x - button_width) / 2
+			button_y = imgui.get_cursor_pos().y + image_height - 50
+
+			# Set the cursor position to the calculated center position for buttons
+			imgui.set_cursor_pos((button_x, button_y))
+
+			# Add buttons under the logo
+			if imgui.button("Singleplayer", width=button_width, height=button_height):
+				# Handle button 1 click
+				pass
+
+			imgui.set_cursor_pos((button_x, imgui.get_cursor_pos().y + 10))
+
+			if imgui.button("Multiplayer", width=button_width, height=button_height):
+				# Handle button 2 click
+				pass
+
+			imgui.set_cursor_pos((button_x, imgui.get_cursor_pos().y + 10))
+
+			if imgui.button("Play tutorial level", width=button_width, height=button_height):
+				# Handle button 2 click
+				pass
+
+			imgui.set_cursor_pos((button_x, imgui.get_cursor_pos().y + 20))
+
+			if imgui.button("Options", width=button_width, height=button_height):
+				# Handle button 2 click
+				pass
+
+			text = "Copyright MC PY contributors. MC PY is licensed under the MIT."
```

Comment:
```suggestion
			text = "Copyright MCPY contributors. MCPY is licensed under the MIT license."
```

Reply:
Resolved in [de26b9d59a9604287380ab74ed1d0156f4a81dcd](https://github.com/obiwac/python-minecraft-clone/pull/104/commits/de26b9d59a9604287380ab74ed1d0156f4a81dcd).